From: Modestas Vainius <modestas@vainius.eu>
Subject: [PATCH] debian/dbug_autoreplace

Add a default autoreplace which replaces dbug:bugnumber with
the respective Debian BTS URL.

Signed-off-by: Modestas Vainius <modestas@vainius.eu>

---
 src/config/preferences.cpp |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

--- a/src/config/preferences.cpp
+++ b/src/config/preferences.cpp
@@ -122,6 +122,7 @@
     QList<QStringList> defaultList;
     defaultList.append(QStringList() << QStringLiteral("1") << QStringLiteral("o") << QStringLiteral("\\[\\[([^\\s]+)\\]\\]") << QStringLiteral("http://en.wikipedia.org/wiki/Special:Search?go=Go&search=%1"));
     defaultList.append(QStringList() << QStringLiteral("1") << QStringLiteral("o") << QStringLiteral("(BUG:|bug:)([0-9]+)") << QStringLiteral("https://bugs.kde.org/show_bug.cgi?id=%2"));
+    defaultList.append(QStringList() << QStringLiteral("1") << QStringLiteral("o") << QStringLiteral("([Dd][Bb][Uu][Gg]:)(\\w+)") << QStringLiteral("https://bugs.debian.org/%2"));
     return defaultList;
 }
 
